src/drivers/code_hosting/activate.sh

```diff
@@ -14,6 +14,10 @@ function activate_driver_for_code_hosting {
     activate_driver 'code_hosting' 'github'
   elif [ "$origin_hostname" == 'bitbucket.org' ]; then
     activate_driver 'code_hosting' 'bitbucket'
+  elif [ "$(contains_string "$origin_hostname" github)" == true ]; then
```

Comment:
These things are not mutually exclusive. We can try to read the SSH identities, and if that works great. If not, we can still do the heuristic approach that I'm adding here. That's another ticket in my opinion, which would need more research and development. I think for the time being here this is not very elegant, but pragmatic. And it doesn't make things worse, since one cannot use Git Town with custom Git remotes at all right now.